server.go

```diff
@@ -471,6 +472,13 @@ type server struct {
 	bytesSent     uint64 // Total bytes sent by all peers since start.
 	shutdown      int32
 
+	// minKnownWork houses the minimum known work from the associated network
+	// params converted to a uint256 so the conversion only needs to be
+	// performed once when the server is initialized.  Ideally, the chain params
```

Comment:
nit: rm extra space here

Reply:
Pretty much all comments in dcrd use double spaces after periods.  This is one of those things like 80 cols that can be a point of contention.  See https://link.springer.com/article/10.3758/s13414-018-1527-6.